vite.plugins.ts

```diff
@@ -4,24 +4,25 @@ import { minify } from "html-minifier-terser";
 import { extname } from "path";
 import { Plugin } from "vite";
 import viteCompression from "vite-plugin-compression";
-
 /**
- * Read the II canister ID from dfx's local state
+ * Read a canister ID from dfx's local state
  */
-const readCanisterId = (): string => {
-  const canisterIdsJsonFile = "./.dfx/local/canister_ids.json";
-
+export const readCanisterId = (
```

Comment:
suggestion: name the arguments to avoid issues like this:

```
readCanisterId(
         "./.dfx/local/canister_ids.json",
         "internet_identity",
       )
```